src/css/base.scss

```diff
@@ -292,8 +292,8 @@ h5 {
 }
 
 .article__widget--div span {
```

Comment:
.all-time-sleep, this .article_widget-div,  and .steps_article--3 would be an great chance to use a mixin, as we could just change the value of the font-size and everything else is the same.